lib/testOptions.ts

```diff
@@ -0,0 +1,60 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as path from 'path';
+
+/**
+ * Get the default mocha options for ads extension unit test. The following environment variables will be examined.
+ * ADS_TEST_GREP: custom test filter.
+ * ADS_TEST_INVERT_GREP: whether to invert the grep results.
+ * RUN_UNSTABLE_TESTS: whether to run the unstable tests. By default, only the stable tests will be executed unless this variable is set to true. If ADS_TEST_GREP is specified, this variable will be ignored.
+ * ADS_TEST_TIMEOUT: test timeout.
+ * ADS_TEST_RETRIES: number of retries.
+ * BUILD_ARTIFACTSTAGINGDIRECTORY: target directory for the test reports.
+ * @param testSuiteName The test suite name.
+ */
+export function getDefaultMochaOptions(testSuiteName: string): any {
+	const mochaOptions: any = {
```

Comment:
It'd be good to have actual types here - you can just use `@types/mocha` to get those without bringing in the entire library.

Reply:
done, found the typings version that has the invert option defined in the options.